### PR TITLE
Feat: Question, result

### PR DESCRIPTION
**Pull request by yashpatel1797**
Feature:
Questions list
result page

### CODE REVIEW

File: src/backend/controllers/QuizesController.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ export const getAllQuizesHandler = function () {
 
 export const getSingleQuizHandler = function (schema, request) {
   const Id = request.params.quizId;
+  console.log(Id);
```
Comment (ravikumar0403) [inline src/backend/controllers/QuizesController.js]: remove logs

**Review by ravikumar0403 — APPROVED** [review]
Code is clean, 
good folder structure
Good use of react portals

File: src/pages/Result/Result.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    const calculateScore = (quiz, answers) => {
+        let score = 0;
+        score = quiz.reduce((acc, curr) =>
+            curr.options.find(({ _id }) => _id === answers[curr._id])?.isCorrect ? acc + 10 : acc, score);
+        return score;
+    }
+
+    return (
+        <>
+            <div className={`${styles.container} container`}>
```
Comment (rohanmathur91) [inline src/pages/Result/Result.js:19]: no need of fragment over here

File: src/reducer/quizReducer.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            return {
+                ...state,
+                quizId: payload.quizId,
+                answers: {
+                    ...state.answers,
+                    [payload.questionId]: payload.selectedAnswer,
+                },
+            }
+        default:
+            return state;
```
Comment (rohanmathur91) [inline src/reducer/quizReducer.js:17]: throw an error saying `action type not found` or something similar to this

**Review by rohanmathur91 — APPROVED** [review]
- liked the use of CSS modules
- good way to provide an abstraction for toast-container by wrapping it into other component
- like the way you have handled the routes

**Pull request merged**